### PR TITLE
Update weapons and attachments to implement military flashlight attachment

### PR DESCRIPTION
**Pull request by Streatman**

### CODE REVIEW

File: SQF/dayz_code/stringtable.xml [hunk 1]
```diff
@@ -9772,6 +9772,11140 @@
 			<English>M4A1 FL</English>
 			<Russian>M4A1 Ф</Russian>
 		</Key>
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml:9774]: duplicates

**Review by ebayShopper — CHANGES_REQUESTED** [review]

File: SQF/dayz_code/stringtable.xml (hunk 1 above)
Comment (Streatman) [reply SQF/dayz_code/stringtable.xml:9774]: oh.... oups

**Review by Streatman — COMMENTED** [review]
i fixed the stringtables and added the remington that i forgot

File: SQF/dayz_code/stringtable.xml [hunk 2]
```diff
@@ -10941,6 +11112,18 @@
 			<German>Entferne Taschenlampe</German>
 		</Key>
 		
+		<!-- Military pistol flashlight -->
+		<Key ID="STR_DZ_ATT_MFL_PST_NAME">
+			<English>[A] Military pistol flashlight</English>
+			<Russian>[М] военный пистолетный фонарь</Russian>
+			<German>[E] Militärische Pistolen Taschenlampe</German>			
+		</Key>
+		<Key ID="STR_DZ_ATT_FL_PST_DESC">
```
Comment (ebayShopper) [inline SQF/dayz_code/stringtable.xml]: should be `STR_DZ_ATT_MFL_PST_DESC`

Comment (Streatman) [reply SQF/dayz_code/stringtable.xml]: fixed

**Pull request merged**